### PR TITLE
Android standalone toolchains are not longer necessary

### PR DESCRIPTION
**Pull request by romainguy**
PLEASE CLEAN YOUR BUILD DIRECTORY (`rm -Rf out/`).

The latest NDK (19) contains ABI/API level specific command line
tools making standalone toolchains unnecessary. This change adapts
to the new model which greatly simplifies the build script and
ensures the latest version of the tools is being used.

Note that this requires a fairly recent version of CMake which fixes
a bug related to ranlib. This change was tested with CMake 3.13
(CMake 3.7 does not work).

### CODE REVIEW

**Comment by romainguy** [comment]
Breaking in CI because of the version of CMake.

File: README.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ and tools.
 
 To build Filament, you must first install the following tools:
 
-- CMake 3.4 (or more recent)
+- CMake 3.13 (or more recent)
```
Comment (bejado) [inline README.md]: Maybe we could relax this requirement for users not building Android?

Comment (romainguy) [reply README.md]: I thought about it but wanted to keep things simple.

Comment (rawnsley) [reply README.md]: I think the Android build should check for this requirement explicitly and fail with a useful message. The build errors I got were very opaque with cmake 3.10 and I took a circuitous route to get here.

Comment (romainguy) [reply README.md]: build.sh now makes the check.

**Pull request merged**